operations/power_management/Power_On_and_Start_the_Management_Kubernetes_Cluster.md

```diff
@@ -286,54 +295,31 @@ Verify that the Lustre file system is available from the management cluster.
     ----------------
     ncn-m[001-003]
     ----------------
+    /etc/cray/upgrade/csm is not a mountpoint
     /var/opt/cray/sdu/collection-mount is a mountpoint
     /var/opt/cray/config-data is a mountpoint
     ----------------
     ncn-w[001-003]
     ----------------
+    /etc/cray/upgrade/csm is a mountpoint
     /var/lib/cps-local/boot-images is a mountpoint
     ```
 
-1. (`ncn-m001#`) Monitor the status of the management cluster and which pods are restarting (as indicated by either a `Running` or `Completed` state).
+1. (`ncn-m001#`) Correct the SDU collection link.
 
-    ```bash
-    kubectl get pods -A -o wide | grep -v -e Running -e Completed
-    ```
-
-    The pods and containers are normally restored in approximately 10 minutes.
-
-    Because no containers are running, all pods first transition to an `Error` state. The error state indicates that their containers were stopped. The `kubelet` on each node
-    restarts the containers for each pod. The `RESTARTS` column of the `kubectl get pods -A` command increments as each pod progresses through the restart sequence.
-
-    If there are pods in the `MatchNodeSelector` state, delete these pods. Then verify that the pods restart and are in the `Running` state.
-
-1. (`ncn-m001#`) Check the status of the `slurmctld` and `slurmdbd` pods to determine if they are starting:
+    If SDU has been configured on a master node, correct the `collection` link now that its `fuse.s3fs` filesystem is mounted from Ceph storage nodes.
+    Change the link target of `/var/opt/cray/sdu/collection` to `collection-mount` instead to have `collection-local` as the link target. This command checks all master nodes but changes them and restarts `cray-sdu-rda` only if needed.
```

Comment:
Looks like extra words were left in after an edit.

```suggestion
    Change the link target of `/var/opt/cray/sdu/collection` to `collection-mount` instead of `collection-local`. This command checks all master nodes but changes them and restarts `cray-sdu-rda` only if needed.
```

Reply:
fixed